libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/FacebookEvents/FacebookResponseEvent.cs

```diff
@@ -6,6 +6,9 @@
 
 namespace Microsoft.Bot.Builder.Adapters.Facebook.FacebookEvents
 {
+    /// <summary>
+    /// Represents in the incoming object received from Facebook and processed by the adapter.
```

Comment:
>    /// Represents in the [](start = 0, length = 25)

Should it be "Represents the incoming..."?

Reply:
Thanks. Will update.